src/sentry/taskworker/registry.py

```diff
@@ -126,13 +126,28 @@ def wrapped(func: Callable[P, R]) -> Task[P, R]:
         return wrapped
 
     def send_task(self, activation: TaskActivation, wait_for_delivery: bool = False) -> None:
-        metrics.incr("taskworker.registry.send_task", tags={"namespace": activation.namespace})
+        metrics.incr(
+            "taskworker.registry.send_task",
+            tags={"namespace": activation.namespace},
```

Comment:
Should we have `taskname` as a tag on this metric as well? We currently emit `sentry.jobs.started` with a task tag (`instance`) and having an analogous metric for taskbroker would be useful.